installation/local/bootstrap.sh

```diff
@@ -65,45 +66,62 @@ echo "1" | sudo update-alternatives --config mpi 1>/dev/null 2>&1 # --> choosing
 echo "1" | sudo update-alternatives --config mpirun 1>/dev/null 2>&1 # --> choosing mpirun
 
 #
-# STEP 2.4 - install NEST Desktop (and dependencies for NEST Server)
+# STEP 3 - install NEST Desktop (and dependencies for NEST Server)
 #
-sudo python3 -m pip install nest-desktop==3.2
-# sudo python3 -m pip install restrictedPython gunicorn flask flask-cors
+pip install nest-desktop==3.3.0a1 \
+	flask flask-cors gunicorn
 
-#
-# STEP 3 - install python packages for the TVB-NEST use-case
-#
 #
 # STEP 4 - TVB
 #
 # NOTE: Specific versions are required for some packages
 pip install --no-cache --target=${CO_SIM_SITE_PACKAGES} \
         tvb-contrib==2.2 tvb-data==2.0 tvb-gdist==2.1 tvb-library==2.2 \
-        cython elephant mpi4py numpy==1.23 pyzmq requests testresources \
-	flask flask-cores gunicorn
+        cython elephant mpi4py numpy==1.23 pyzmq requests testresources
+
+#
+# STEP 5 - Install Insite
+#
+sudo apt-get install -y libssl-dev
+mkdir -p $CO_SIM_INSITE
+cd $CO_SIM_INSITE
+git clone --recurse-submodules https://github.com/VRGroupRWTH/insite.git
+
+cd insite
+
+mkdir build_access_node
+cd build_access_node
```

Comment:
We might want to either always have:
```
mkdir xyz
cd xzy
```
or
```
mkdir xyz; cd xyz 
```
just to be consistent. 
c.f. Line 120